recipe/meta.yaml

```diff
@@ -42,8 +42,8 @@ test:
         - test -f ${PREFIX}/lib/libdf.dylib  # [osx]
         - test -f ${PREFIX}/lib/libmfhdf.so  # [linux]
         - test -f ${PREFIX}/lib/libmfhdf.dylib  # [osx]
-        - conda inspect linkages -n _test hdf4  # [not win]
-        - conda inspect objects -n _test hdf4  # [osx]
+        - conda inspect linkages -p ${PREFIX} hdf4  # [not win]
+        - conda inspect objects -p ${PREFIX} hdf4  # [osx]
```

Comment:
BTW, if you are interested in this feedstock, consider adding yourself as a maintainer :smile: